masq_lib/src/multi_config.rs

```diff
@@ -949,6 +1037,40 @@ pub mod tests {
         assert_eq!(subject.args(), command_line);
     }
 
+    #[test]
+    fn command_line_vcl_return_value_from_vcl_args_by_name() {
+        let command_line: Vec<String> = vec![
```

Comment:
Isn't there a utility function with a long name that you could use in place of this? Something like `vec_of_slices_to_vec_of_strings()` or something?